tests/time_tests/test_runner/conftest.py

```diff
@@ -96,3 +107,43 @@ def get_dict_values(d):
     values = list(get_dict_values(val))
 
     return "-".join(["_".join([key, val]) for key, val in zip(keys, values)])
+
+
+@pytest.mark.hookwrapper
+def pytest_runtest_makereport(item, call):
+    """Pytest hook for report preparation.
+
+    Report tests' data to DB.
+    """
+
+    KEY_FIELDS = ('test_name', 'executable', 'model', 'device', 'niter',
```

Comment:
key fields should be
- `test` - an executable name without extension
- 'test_params' - a concatenation of executable parameters
- `run_id` - build URL or any unique run identifier

Reply:
Done. Please, review once again